tab_pastevents.md

```diff
@@ -12,13 +12,35 @@ tags: cincinnati
 ## Past Events
 <hr>
 
+## November 2022 Meeting
+
+### Security Champions Program
+
+**Discussion Abstract:** A security champions program is unique to the organization and their development teams. As application security teams begin to look towards these types of programs, it's important to have a good understanding of the complexities and goals a program like this will have, along with the relationship your security team has with your development team. In this talk we'll go through what growing and maintaining an effective program looks like, along with some of obstacles and ways to overcome them.
+
+**Speaker Biography:** Jay Simmons is a senior application security analyst at Great American Insurance Group who is responsible for their security champions program. 
+
+**Presentation:*
+[OWASP Cinci - November 2022 - Security Champions Program](https://owasp.org/www-chapter-cincinnati/assets/presentations/Maintaining%20a%20Security%20Champions%20Program.pdf)
+
+## June 2022 Meeting
+
+### Conjur Secrets Management
+
+**Discussion Abstract:** The process of making our code more secure has many components, but how we secure the keys to our kingdom is probably the area with the most to lose. After all, if an attacker has your database credentials, you're toast. Yet many developers don't have a comprehensive system for managing such sensitive information.  Do your software developers have access to the production database? If an employee quits, could they take their credentials with them, or could you easily revoke access without affecting the other developers or production deployments? Can you easily audit access to your secrets and see which applications are using them? Can you easily rotate secrets that are used in containerized deployments in multiple different clouds?  In this talk we'll demonstrate how to solve all of these problems with Conjur, an open source secrets management application.
+
+**Speaker Biography:** Hi! My name is Shlomo Heigh and I'm a senior software engineer at CyberArk. I maintain the Conjur open source secrets management application, primarily working on integrations with platforms like Kubernetes and engaging with the open source community.  Before that, I was a full stack developer for a SaaS firm. I've worked with everything from Ruby to Go and am always looking for new things to learn.  In my spare time I hang out with my wife and 3-year-old daughter, tinker with my 3D printer, and tend to my garden.
+
+**Presentation:**
+TBD
```

Comment:
Here you go
[Conjur Secrets Management.pdf](https://github.com/OWASP/www-chapter-cincinnati/files/10723062/Conjur.Secrets.Management.pdf)

Reply:
Updated pull request to include this document.